src/platform/utilities/cerner/index.js

```diff
@@ -33,9 +33,7 @@ export const getCernerURL = path => {
     ? 'https://patientportal.myhealth.va.gov'
     : 'https://staging-patientportal.myhealth.va.gov';
 
-  return `${host}/clear-session?to=${encodeURIComponent(
-    `${host}${path}?authenticated=true`,
-  )}`;
+  return `${host}${path}?authenticated=true`;
```

Comment:
Just curious but why is the path not encoded anymore?

Reply:
The encoded path was necessary because it was a query parameter being passed to the `clear-session` script. The purpose of this PR effectively boils down to no longer needing the `clear-session` functionality, so, instead, the user is sent directly to the URL that was previously encoded.